base/shell/explorer/settings.cpp

```diff
@@ -93,4 +92,17 @@ BOOL TaskbarSettings::Load()
     return TRUE;
 }
 
+BOOL TaskbarSettings::UseCompactTrayIcons()
+{
+    switch (dwCompactTrayIcons)
+    {
+        case 1:
+            return FALSE;
+        case 2:
```

Comment:
Maybe these "0", "1", "2" values could be replaced with #define'd values or enumerated values. (This would allow them to convey more or less some explanation about their purpose.)